core/src/test/java/org/infinispan/security/ClusteredSecureCacheTest.java

```diff
@@ -30,9 +51,16 @@ public CacheMode getCacheMode() {
    protected void createCacheManagers() throws Throwable {
       final GlobalConfigurationBuilder global = GlobalConfigurationBuilder.defaultClusteredBuilder();
       final ConfigurationBuilder builder = getDefaultClusteredCacheConfig(getCacheMode());
-      global.security().authorization().enable()
-            .principalRoleMapper(new IdentityRoleMapper()).role("admin").permission(AuthorizationPermission.ALL);
-      builder.security().authorization().enable().role("admin");
+      GlobalAuthorizationConfigurationBuilder globalRoles = global.security().authorization().enable()
+            .principalRoleMapper(new IdentityRoleMapper());
+      for (AuthorizationPermission perm : AuthorizationPermission.values()) {
+         globalRoles.role(perm.toString()).permission(perm);
+      }
+      global.serialization().addAdvancedExternalizer(4321, new SecureConsumer.Externalizer());
```

Comment:
Why are we using an Externalizer here instead of a ProtoAdapter?

Reply:
For the test, I thought it was quicker